### PR TITLE
Sign up card added

### PR DESCRIPTION
**Pull request by anandsr21harshit**
image

### CODE REVIEW

File: signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.login-card{
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 6px -1px, rgba(0, 0, 0, 0.06) 0px 2px 4px -1px;
+    width: 22rem;
+    padding: 0.5rem;
+    border: 2px solid rgba(0, 0, 0, 0.1);
+}
+
+#userName{
```
Comment (surajmore98) [inline signup/signup.css:49]: try to use class instead of id.

File: signup/signup.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+.input-container{
+    margin: 2rem 0rem;
+}
+
+.input-container a{
+    color: inherit;
+    text-decoration: none;
+}
+
+#conditions{
```
Comment (surajmore98) [inline signup/signup.css:25]: try to use class instead of id

**Review by shrey27 — APPROVED** [review]

File: signup/signup.css (hunk 1 above)
Comment (anandsr21harshit) [reply signup/signup.css:49]: I know id are less preferred. But this style is specific to those only. I don't need them anywhere else.

Comment (surajmore98) [reply signup/signup.css:49]: ok no issue then

**Review by surajmore98 — APPROVED** [review]

**Pull request merged**